core/src/main/scala/sttp/tapir/Validator.scala

```diff
@@ -46,9 +46,43 @@ object Validator extends ValidatorMacros {
 
   // string
   def pattern[T <: String](value: String): Validator.Primitive[T] = Pattern(value)
-  def minLength[T <: String](value: Int): Validator.Primitive[T] = MinLength(value)
-  def maxLength[T <: String](value: Int): Validator.Primitive[T] = MaxLength(value)
-  def fixedLength[T <: String](value: Int): Validator[T] = MinLength(value).and(MaxLength(value))
+
+  /** Create a validator for minimum length constraints on strings.
+    *
+    * @param value
+    *   The minimum allowed length.
+    * @param countCodePoints
+    *   A boolean parameter that determines whether the validation will consider code points or character count. When set to true, the
+    *   validator will consider characters that are represented using two code units (a surrogate pair) in UTF-16 encoding allowing for a
+    *   more accurate length validation for strings containing such characters. Defaults to false, where length is validated based on the
+    *   number of `Char` values in the string.
+    */
+  def minLength[T <: String](value: Int, countCodePoints: Boolean = false): Validator.Primitive[T] = MinLength(value, countCodePoints)
+
+  /** Create a validator for maximum length constraints on strings.
+    *
+    * @param value
+    *   The maximum allowed length.
+    * @param countCodePoints
+    *   A boolean parameter that determines whether the validation will consider code points or character count. When set to true, the
+    *   validator will consider characters that are represented using two code units (a surrogate pair) in UTF-16 encoding allowing for a
+    *   more accurate length validation for strings containing such characters. Defaults to false, where length is validated based on the
```

Comment:
**nitpick:** "such characters" -> "such characters (like emojis)", maybe such a hint will additionally help people to understand if this is their case.